experiments/benchmarks/benchmark_hypersphere.py

```
@@ -33,7 +33,7 @@ def benchmark_hypersphere(d, r=0.5, n_batch=100000, lr=1.e-3):
     }
     integrand = RegulatedHyperSphereIntegrand(d, device=device, **integrand_params)
     optim = partial(torch.optim.Adam, lr=lr)
-    integrator = Integrator(f=integrand, d=d, device=device, trainer_options={"minibatch_size": 20000, "optim": optim})
+    integrator = Integrator(f=integrand, d=d, device=device, trainer_options={"minibatch_size": 20000, "optim": optim,"masking":"iflow"})
```

remove and use get_default_integrator_config and override the masking entry